examples/mysql-wordpress-pd/README.md

```diff
@@ -278,6 +326,10 @@ NAME        CLUSTER-IP     EXTERNAL-IP     PORT(S)   AGE
 wordpress   10.0.0.5       1.2.3.4         80/TCP    19h
 ```
 
+**Note:** if you're using Host Path for storage then the wordpress service's
+`LoadBalancer` type is not implemented, and thus the wordpress pod's external
+IP address is not available (remains "Pending").
+
 ## Visit your new WordPress blog
```

Comment:
Remove this please. The prerequisites at the top require a Load Balancer. This is not related to hostpath.